### ISSUE TITLE
cryptographic error

### ISSUE THREAD

**Issue by westaking** [iqlusioninc/tmkms#24]
tmkms can't access desmos v0.3.0 for validator.
also it can't access regen-network wasmd v0.7.1

Perhaps tmkms doesn't seem to be able to access the project that used for cosmos-sdk v0.38.1.

The URL below has detailed reporting.
https://github.com/desmos-labs/desmos/issues/107

**Comment by tony-iqlusion** [comment]
My best guess is this is due to changes to the Secret Connection handshake which landed in Tendermint v0.33:

https://github.com/tendermint/tendermint/pull/4185

There was previously this PR to add them, although I'd need to check if it's actually compatible to what shipped in 

https://github.com/tendermint/kms/pull/254

**Comment by zmanian** [comment]
yeah now that there is a tendermint release. I should update the code. 

I'll work on this.

**Comment by marcotradenet** [comment]
I tried to connect a node of one of our chain and the tmkms client returns me this error


```
Apr 30 14:50:58.016 ERROR tmkms::client: [chain-YYoRf9@tcp://10.75.25.101:46658] cryptographic error
```

The node does not start and returns this error

```
E[2020-04-30|11:37:55.631] SignerClient::GetPubKey                      module=privval err="endpoint connection timed out"
ERROR: could not retrieve public key from private validator
github.com/tendermint/tendermint/node.NewNode
	/root/go/pkg/mod/github.com/tendermint/tendermint@v0.33.3/node/node.go:624
github.com/cosmos/cosmos-sdk/server.startInProcess
	/root/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.38.3/server/start.go:157
github.com/cosmos/cosmos-sdk/server.StartCmd.func1
	/root/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.38.3/server/start.go:67
github.com/spf13/cobra.(*Command).execute
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:840
github.com/spf13/cobra.(*Command).ExecuteC
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:945
github.com/spf13/cobra.(*Command).Execute
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:885
github.com/tendermint/tendermint/libs/cli.Executor.Execute
	/root/go/pkg/mod/github.com/tendermint/tendermint@v0.33.3/libs/cli/setup.go:89
main.main
	/root/commercionetwork/cmd/cnd/main.go:82
runtime.main
	/snap/go/5646/src/runtime/proc.go:203
runtime.goexit
	/snap/go/5646/src/runtime/asm_amd64.s:1373
```

**Comment by zmanian** [comment]
yeah we don't support tendermint 0.33+ yet.

**Comment by zmanian** [comment]
I'm hoping to work on this during Game of Zones but we will see.

**Comment by tony-iqlusion** [comment]
Fixed in #58
